source/content/guides/drupal/drupal-hosted/01-introduction.md

```diff
@@ -1,46 +1,46 @@
 ---
-title: Migrate a Drupal 8 Site to Drupal 9
+title: Migrate a Drupal 8 Site to the Latest Version of Drupal
```

Comment:
@stovak , should the title be "Migrate a Drupal 8 Site to the Latest Version of Drupal" or "Update to the Latest Version of Drupal"?